.github/workflows/ci.yml

```diff
@@ -11,39 +11,48 @@ env:
   GO_VERSION: ^1.21
   GOLANGCI_LINT_VERSION: v1.54.0
 
+permissions:
```

Comment:
1. why not `permissions: read-all`?
2. what is profit from `permissions ` in specific context of this project?

Reply:
read-all allows way more access than just contents: read.
It's usually better to accord only required permission to actions